di-ipv-evcs-stub/lambdas/src/handlers/evcsHandler.ts

```diff
@@ -0,0 +1,169 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResultV2 } from "aws-lambda";
+import { buildApiResponse } from "../common/apiResponses";
+import PostRequest from "../domain/postRequest";
+import PatchRequest from "../domain/patchRequest";
+import ServiceResponse from "../domain/serviceResponse";
+import PersistVC from "../domain/persistVC";
+import UpdateVC from "../domain/updateVC";
+import { CreateVcStates, UpdateVcStates } from "../domain/enums/vcState";
+
+import { processPostUserVCsRequest } from "../services/evcsService";
+import { processGetUserVCsRequest } from "../services/evcsService";
+import { processPatchUserVCsRequest } from "../services/evcsService";
+import { verifyToken } from "../services/jwtService";
+
+export async function handler(
+  event: APIGatewayProxyEvent,
+): Promise<APIGatewayProxyResultV2> {
+  console.info(`---Request received----`);
+  const userId = event.pathParameters?.userId;
+  if (!userId) {
+    return buildApiResponse({ errorMessage: "Missing userId." }, 400);
+  }
+
+  try {
+    let request;
+    let res: ServiceResponse = {
+      response: Object,
+    };
+    switch (event.httpMethod) {
+      case "POST":
+        try {
+          request = parsePostRequest(event);
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (error: any) {
+          console.error(error);
+          return buildApiResponse({ errorMessage: error.message }, 400);
+        }
+        res = await processPostUserVCsRequest(
+          decodeURIComponent(userId),
+          request,
+        );
+        break;
```

Comment:
Could inline these to tighten it up a bit
```suggestion
    try {
          res = await processPostUserVCsRequest(
              decodeURIComponent(userId),
              parsePostRequest(event))
          // eslint-disable-next-line @typescript-eslint/no-explicit-any
        } catch (error: any) {
          console.error(error);
          return buildApiResponse({ errorMessage: error.message }, 400);
        }
        break;
```

Reply:
the outer try...catch will take carte of that ...to return 500 for any failure frm service side.